front_left_pid:
  p: 1.0
  i: 0.1
  d: 0.01

front_right_pid:
  p: 1.5
  i: 0.2
  d: 0.02

rear_left_pid:
  p: 0.8
  i: 0.05
  d: 0.005

rear_right_pid:
  p: 1.2
  i: 0.15
  d: 0.015

wheelbase: 0.5
track: 0.3
